libultraship/libultraship/Lib/Fast3D/gfx_rendering_api.h

```diff
@@ -21,6 +22,17 @@ enum FilteringMode {
     NONE
 };
 
+namespace std {
```

Comment:
this is a hashing function for a pair of two floats.. different files need this, not sure where the best place to put it would be..

Reply:
Thanks for the suggestion! I've now made this a custom hash function that gets passed into `unordered_map`.